Installing-Oppia.md

```diff
@@ -1,455 +1,35 @@
-## Table of Contents
+__If you are looking for more elaborate instructions on how to get started with Oppia, go to [setting things up](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up).__
+If you get stuck somewhere and need help please refer to our [[Getting Help Page|Get-help]].
 
-* [Install prerequisites](#install-prerequisites)
-* [Clone Oppia](#clone-oppia)
-* [Setup a virtual environment](#setup-a-virtual-environment)
-* [Running Oppia on a development server](#running-oppia-on-a-development-server)
-* [Tips and tricks](#tips-and-tricks)
-* [Notes on installation on Arch Linux systems](#notes-on-installation-on-arch-linux-systems)
-  * [Changes to installation prerequisites](#changes-to-installation-prerequisites)
-  * [Changes to the virtual environment setup](#changes-to-the-virtual-environment-setup)
+Note that throughout our wiki, we use the following notations:
 
-**Note:** If you just want to create and share explorations, you may be able to use the hosted server at https://www.oppia.org (in which case you don't need to install anything).
+* `$` represents a command prompt that is not root. For example, if we want you to type the command `python` into your terminal, you'll see `$ python`. This distinguishes commands (which are prefixed with `$`) from output from those commands (whose lines are not prefixed with `$`).
+* `#` represents a root command prompt. For example, `# apt-get install python` means to run `apt-get install python` as root (e.g. using `sudo`).
+* `...` indicates where we've omitted content for the sake of brevity.
 
-*These installation instructions were last tested on 24 July 2021. For more information on issues that may occasionally arise with the installation process, see the [Troubleshooting](https://github.com/oppia/oppia/wiki/Troubleshooting) page or ask in the GitHub Discussions.*
```

Comment:
I think we should keep this, or something like it?

Reply:
@seanlip We could update the date to 8 Jan 2024, as I personally installed it on that date using these commands. We can also include the link of the GitHub discussion above.

Reply:
Yes that sounds fine to me, thanks.

Reply:
Done!